code/modules/mob/hearing/virtualhearer.dm

```diff
@@ -71,12 +71,12 @@ var/list/stationary_hearers = list(	/obj/item/device/radio/intercom,
 		sight = copying
 	if(adding)
 		sight |= adding
-		if(adding & SEE_TURFS)
+		if((adding & SEE_TURFS) || (adding & SEE_MOBS))
```

Comment:
```suggestion
		if(adding & (SEE_TURFS | SEE_MOBS))
```
Bitwise checks can be combined like this

Reply:
cool, i didn't know that and I made the changes